test/PriceChopper.js

```diff
@@ -0,0 +1,41 @@
+const assert = require("assert");
```

Comment:
Not sure why this test is in this PR. it's fine now but break it out next time please, unless there's a good reason!